### ISSUE TITLE
need a script for pushing KG2 to Neo4j

### ISSUE THREAD

**Issue by saramsey** [RTXteam/RTX#349]
We need a script that can download KG2 in JSON format and then push it to a Neo4j database. Maybe KGX can be used for this?  

Here is the link to KG2:
https://s3-us-west-2.amazonaws.com/rtx-kg2-public/index.html

it is sorta in a biolink-model, obograph-ish JSON format, though I haven't validated it.

**Comment by saramsey** [comment]
To clarify-- the script should create a "new" KG in Neo4j based on an input "kg2.json" file.

**Comment by saramsey** [comment]
@saramsey  to provide two KG JSON files for @finnagin to test out; a "mini" KG with maybe 50,000 nodes and a "mini mini" KG withi 5,000 nodes

**Comment by saramsey** [comment]
@finnagin  here is an example (relatively small, 16 MB) KG2 for testing your script:

https://s3-us-west-2.amazonaws.com/rtx-kg2-public/kg2-test.json.gz

**Comment by saramsey** [comment]
@finnagin  any thoughts about whether KGX can be used for this?

**Comment by finnagin** [comment]
@saramsey I'm not sure. I've got a docker file now that installs kgx but I haven't been able to get the `kgx neo4j-upload` command to work. I've tried with both the master branch and the source-sink branch and each give me different errors. The installer installs the wrong version of 3 dependencies that I had to manually reinstall the correct versions after so it's possible there is something else I'm missing with the installation still.

**Comment by saramsey** [comment]
@finnagin please use this JSON file as the example "mini" KG2:
https://rtx-kg2-public.s3-us-west-2.amazonaws.com/kg2-test.json.gz

**Comment by saramsey** [comment]
Can we convert a TTL file (say, from Monarch) to JSON using `kgx dump` and see what the JSON looks like?  Here is an example TTL file:

https://archive.monarchinitiative.org/201702/ttl/animalqtldb_test.ttl

**Comment by finnagin** [comment]
Deepak got back to me yesterday and is working on a fix for the bug I was running into with kgx: https://github.com/NCATS-Tangerine/kgx/issues/127

**Comment by finnagin** [comment]
I've been able to write some cypher that uses apoc to upload the json if we need to fall back on that. I've been able to get it to run on a small test example now I just need to test the performance on a larger subset.

@saramsey It looks like edge properties cannont be dictionaries so I couldn't include the `publications info` property on the edges as I get the error message `Property values can only be of primitive types or arrays thereof`. Thoughts on what we can change this to?
